README.md

```diff
@@ -17,11 +17,12 @@ tools this project uses. (Don't spend too long on these; just use them to get a
 1. Clone this repository
 2. Get a copy of the initial data:
     - [Excel Sheet](https://drive.google.com/file/d/1xiLeyMwZc-n6eB1R_XdCJ00YrarfnR_w/view)
-    - Download as csv to the project root. In the steps below we refer to this file as `filename.csv`
+    - Download as csv to the project root. Make sure you choose the `CSV (Comma delimited) (*.csv)` option. In the steps below we refer to this file as `filename.csv`
+3. Install [PostgreSQL](https://www.postgresql.org/) if you haven't already.
```

Comment:
This should be in the No Containers section. If using Docker, you do not need to install Postgres yourself.